### ISSUE TITLE
cc-button: implement parts of ARIA API

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#699]
# Context

There are some situations where we need to add ARIA attributes to make a button accessible.
To name a few:

- `aria-expanded` for [disclosure patterns](https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/) (Show / Hide)
- `aria-controls` for the same pattern as above. => we cannot do it yet, we need to wait for `Cross root ARIA - export ID` (https://github.com/WICG/aom/blob/gh-pages/exportid-explainer.md)
- `aria-pressed` for toggle button patterns.

The only way to do this is to mimic the ARIA API by adding props to our `cc-button`.
We'll need to make sure these props are properly namespaced to avoid clashing with actual ARIA.

(Once Accessibility Object Model is available :roll_eyes: we'll be able to simplify this but we cannot really wait for it)
